dev-qt/qtbase/qtbase-6.7.2-r1.ebuild

```diff
@@ -12,7 +12,7 @@ if [[ ${QT6_BUILD_TYPE} == release ]]; then
 fi
 
 declare -A QT6_IUSE=(
-	[global]="+ssl +udev zstd"
+	[global]="+ssl +udev zstd systemd syslog"
```

Comment:
I'd personally add a local metadata.xml entry for both flags so that can describe the caveats as noted in previous comment. The global description, e.g. `Enable support for syslog` doesn't say much.

Albeit I could add/reword that myself later, so don't worry too much about it.

Reply:
Hm, actually that example made me realize syslog is a global flag (kind of slipped my mind), and made me wonder if we should rename it too so it's not enabled by accident too easily.. but it's used by a total of 1 ebuild currently and not a profile default unlike USE=systemd, so guess it's fine.

Guess *could* consider USE="logger-journald logger-syslog" if we really wanted to differentiate, albeit does feel kind of redundant when journald and syslog are kind of self-explanatory as being for logging.

Reply:
I added metadata.xml entries for both flags.